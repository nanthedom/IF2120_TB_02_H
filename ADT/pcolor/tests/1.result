ADT/pcolor/tests/1.in ADT/pcolor/tests/1.out: TRUE
