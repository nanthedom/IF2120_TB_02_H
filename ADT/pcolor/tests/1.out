[31mR[0m[32mG[0m[34mB[0m
